history-of-the-quakers-v4/original/18-book-7-chapter-12.adoc

```diff
@@ -285,6 +289,10 @@ nominated twelve friends as representatives of the yearly meeting,
 and directed the respective Quarterly meetings to
 appoint four members each to represent them.
 
+[.embedded-content-document.letter]
+--
+
+[.letter-heading]
 The Services to be performed by the Meeting for Sufferings, viz.
```

Comment:
Wasn't sure about this.

Reply:
Yeah, I don't think this is a letter.  I sort of lean towards leaving off a type, and then changing line 296 to just be a plain sub-heading, not a letter heading.